docs/index.md

```diff
@@ -19,6 +19,10 @@ manages PostgreSQL clusters on Kubernetes:
 
 3. Finally, the operator periodically synchronizes the actual state of each
    Postgres cluster with the desired state defined in the cluster's manifest.
+   This is triggered by either the `sync scan`, running every `resync_period`
+   seconds for every cluster, or by the `repair scan`, coming every
+   `repair_period` only for those clusters that didn't report success as a
+   result of the last operation running on them.
```

Comment:
This paragraph  is better fitted for the administrator docs; I wrote the Intro with the intention to provide a very high-level overview of operator's capabilities w/o any overly technical details.

Reply:
Good point, moved to the end of the admin guide